crates/astria-core/src/sequencer/v1/abci.rs

```diff
@@ -17,6 +17,7 @@ impl AbciErrorCode {
     pub const INTERNAL_ERROR: Self = Self(3);
     pub const INVALID_NONCE: Self = Self(4);
     pub const TRANSACTION_TOO_LARGE: Self = Self(5);
+    pub const INSUFFICIENT_FUNDS: Self = Self(6);
```

Comment:
Should there be a matching ABCI error code added here? https://github.com/astriaorg/astria/blob/main/proto/sequencerapis/astria/sequencer/v1/abci.proto#L5

Reply:
turns out these are unused, so i removed them

Reply:
ugh this is going to break the protos in ci even though these were unused :/ let me know if that's ok